edition = "2023";

package xls;

option java_multiple_files = true;

message PassDocumentationProto {
  message OnePass {
    string short_name = 1;
    string long_name = 2;
    string file = 3;
    string notes = 4;
  }
  repeated OnePass passes = 1;
}
